config/deploy.rb

```diff
@@ -44,11 +44,12 @@
       end
     end
   end
-  
+
   desc "Restart zartan-related monitored processes"
   after :seed_config, :restart do
     on roles(:web) do
-      if test("ps cax | grep monit")
+      enabled_file = File.join(fetch(:deploy_to), 'shared', 'enabled')
```

Comment:
What creates the file `shared/enabled`? This isn't related to `GOOGLE_OMNIAUTH["enabled"]`, is it?